src/main/java/org/primefaces/component/picklist/PickListBase.java

```diff
@@ -99,6 +100,14 @@ public void setVar(String var) {
         getStateHelper().put(PropertyKeys.var, var);
     }
 
+    public String getAllowDuplicates() {
```

Comment:
This should return `boolean` and be `isAllowDuplicates`

Reply:
My bad. Fixed.